lib/cryptography.js

```diff
@@ -28,7 +29,12 @@ module.exports.decrypt = (options) => {
     const decipher = crypto.createDecipheriv(decryptionAlgo, key, iv)
     let decrypted = decipher.update(ciphertext, 'hex', 'utf8')
     decrypted += decipher.final('utf8')
-    return decrypted
+    if (fs.existsSync(outputFilePath)) {
+      fs.copyFileSync(outputFilePath, `${outputFilePath}.bak`)
+    }
+    fs.writeFileSync(outputFilePath, decrypted)
```

Comment:
Please note that this same function is used in https://github.com/kunalpanchal/secure-env/blob/master/lib/index.js

so we should not create or write any file if its used within the code.

All of this should only happen if the outputFilePath exists. Otherwise just log it the screen. If -o option is not sent via the CLI